fail2ban/tests/utils.py

```diff
@@ -368,6 +448,38 @@ def assertNotIn(self, a, b, msg=None):
 
 class LogCaptureTestCase(unittest.TestCase):
 
+	class _MemHandler(logging.StreamHandler):
+		def __init__(self):
+			self._recs = list()
+			self._strm = StringIO()
+			logging.StreamHandler.__init__(self, self._strm)
+			# 
```

Comment:
I could find at least 10 such code pices, but I suspect it will be a large discussion, so I'll replace it with space.